.travis.yml

```diff
@@ -19,19 +19,13 @@ install:
 
   # Create the basic testing environment
   # ------------------------------------
-  - conda config --set always_yes yes --set changeps1 no
-  - conda config --set show_channel_urls True
+  - conda config --set always_yes yes --set changeps1 no --set show_channel_urls True
   - conda update --quiet conda
+  - conda config --add channels conda-forge
   - ENV_NAME='test-environment'
-  - conda create --quiet -n $ENV_NAME python=$TRAVIS_PYTHON_VERSION
+  - conda create --name $ENV_NAME python=$TRAVIS_PYTHON_VERSION udunits2 --file requirements.txt --file requirements-dev.txt
```

Comment:
Building the env in one go saves us some trouble. Sometimes `conda` does not respect channel preference when adding packages to an env.